src/02_spec/07_modules/cdm/dbgregisters.rst

```diff
@@ -0,0 +1,229 @@
+Programmer Interface: Control Registers
+---------------------------------------
+
+The Control Debug Module implements the :ref:`sec:spec:api:base_register_map`.
+The reset values are to be decided.
+
+.. tabularcolumns:: |p{\dimexpr 0.20\linewidth-2\tabcolsep}|p{\dimexpr 0.20\linewidth-2\tabcolsep}|p{\dimexpr 0.40\linewidth-2\tabcolsep}|p{\dimexpr 0.20\linewidth-2\tabcolsep}|
+.. flat-table:: CDM base register reset values
+  :widths: 2 2 4 2
+  :header-rows: 1
+
+  * - address
+    - name
+    - description
+    - reset value
+
+  * - 0x0000
+    - ``MOD_VENDOR``
+    - module vendor
+    - ?
```

Comment:
Pick the OSD vendor (1), see here http://opensocdebug.readthedocs.io/en/latest/05_idregistry/index.html